src/server/seeds/development/02_cvs.js

```diff
@@ -9,22 +9,22 @@ exports.seed = function(knex) {
           'id': '1',
           'title': 'FrontEnd Developer',
           'description': 'Frontend developer with 3 years experience',
-          'file_url': 'https://zety.com/blog/banking-resume-example',
+          'file_url': 'http://www.africau.edu/images/default/sample.pdf',
```

Comment:
I don't think these needs to be changed?

Reply:
but, i shared a post in slack before changing it. you and Daniel said ok to change. anyway if you want me to undo the commit then i'll revert it.

Reply:
Ah sorry, that was before i learned of @bytewiz nice trick. :)

Reply:
but if i keep original them am getting this
![pdf](https://user-images.githubusercontent.com/66561373/107428971-a61edd00-6b23-11eb-8c0d-13ca4f68319d.PNG)

Reply:
Okay, then it's fine. For future reference, i had to click a little and ended up with this one: 

blob:https://www.wozber.com/21cc7cd6-c62f-4b34-aec4-06d746f3e116